keyboards/consolekeyboard/rules.mk

```diff
@@ -0,0 +1,37 @@
+# MCU name
+MCU = atmega32u4
+
+# Processor frequency.
+#     This will define a symbol, F_CPU, in all source code files equal to the
+#     processor frequency in Hz. You can then use this symbol in your source code to
+#     calculate timings. Do NOT tack on a 'UL' at the end, this will be done
+#     automatically to create a 32-bit value in your source code.
+#
+#     This will be an integer division of F_USB below, as it is sourced by
+#     F_USB after it has run through any CPU prescalers. Note that this value
+#     does not *change* the processor frequency - it should merely be updated to
+#     reflect the processor speed set externally so that the code can use accurate
+#     software delays.
+#F_CPU = 8000000
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
```

Comment:
> Hardware Supported: Arduino Pro Micro

Readme mentions pro micro, so that would usually be `BOOTLOADER = caterina`

Reply:
i havent tried using caterina but the promicro im using is a 3v3 variant (hence commented out 8mhz line)